### PR TITLE
Header footer

### PR DESCRIPTION
**Pull request by AndrewBestbier**
### Motivation
[Related story links?](ticket, design)

### Test plan
- Click x in all supported browsers and y should happen

### Pre-flight checklist
- [ ] Unit tests
- [ ] Has been design reviewed
- [ ] Add a gif - `gif`

### CODE REVIEW

File: site/components/footer/style.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+.nav li {
+  margin: 0 0 20px 0;
+}
+
+.nav li:last-child {
+  margin: 0;
+}
+
+.nav li a {
+  border-bottom: 2px solid red;
```
Comment (zoeabryant) [inline site/components/footer/style.css]: change to `composes: redUnderlineLink from "../../css/_links.css";`

File: site/components/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+              href="https://github.com/redbadger"
+              title="Red Badger Youtube"
+              className={styles.socialIcon}
+            >
+              <InlineSVG src={youtubeSVG} />
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://github.com/redbadger"
```
Comment (zoeabryant) [inline site/components/footer/index.js]: All these links go to github. Please change

File: site/components/footer/index.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    </div>
+    <div className={cx('section', 'disclaimer', 'noBorder')}>
+      <p className={styles.afterDivider}>&copy; Red Badger Consultancy Ltd 2016</p>
+      <p className={styles.afterDivider}>Registered number 345 678 912 UK</p>
+      <p>VAT Registration No. 990 8085 82</p>
+      <p className={styles.cookieWarning}>
+        We use cookies on our website. For more information, view our privacy policy.
+      </p>
+    </div>
+    <InlineSVG src={badgerSVG} className={styles.badgerIcon} />
```
Comment (zoeabryant) [inline site/components/footer/index.js]: Badger should be clickable - to go to homepage

File: site/css/_links.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.redUnderlineLink {
+  border-bottom: 2px solid red;
+  color: white;
+  text-decoration: none;
+}
```
Comment (zoeabryant) [inline site/css/_links.css]: please add an example like _colors and _breakpoints, I believe they are still considered useful by the team.

Comment (lpil) [inline site/css/_links.css]: Should we always use variables for colours?

Comment (zoeabryant) [inline site/css/_links.css]: yes - any variants should be double checked with the designer

**Pull request merged**